analyzers/tests/Directory.Build.props

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
 
+  <!-- Import the higher level common properties -->
+  <Import Project="..\Directory.Build.props" />
```

Comment:
We need to reference parent the `Directory.Build.props` in tests, otherwise it is not considered and the configuration is not applied.